commands/mr/create/mr_create.go

```diff
@@ -329,9 +334,9 @@ func NewCmdCreate(f *cmdutils.Factory) *cobra.Command {
 					message = "\nCreating draft merge request for %s into %s in %s\n\n"
 				}
 
-				fmt.Fprintf(opts.IO.StdErr, message, utils.Cyan(opts.SourceBranch), utils.Cyan(opts.TargetBranch), repo.FullName())
+				fmt.Fprintf(opts.IO.StdErr, message, utils.Cyan(opts.SourceBranch), utils.Cyan(opts.TargetBranch), baseRepo.FullName())
 
-				mr, err := api.CreateMR(labClient, repo.FullName(), mrCreateOpts)
+				mr, err := api.CreateMR(labClient, headRepo.FullName(), mrCreateOpts)
```

Comment:
The merge request is created in the `baseRepo` instead

```suggestion
				mr, err := api.CreateMR(labClient, baseRepo.FullName(), mrCreateOpts)
```

Reply:
Can be either really as long as SourceProjectID and TargetProjectID point to the right places, but yes makes more sense to have it against the target repo.

Reply:
pushed with `headRepo` instead

Reply:
Yeah it is not ok to use `baseRepo`, it fails on cross-repository branches, the merge request should be from the repository where the source branch is.

so it should be created always with `headRepo`

Reply:
Merge Request created against `Leo/aports`: https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/16206
Merge Requests created against `alpine/aports`:
    https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/16205
    https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/16204